samples/subsys/ipc/openamp_rsc_table/prj.conf

```diff
@@ -1,11 +1,12 @@
 CONFIG_KERNEL_BIN_NAME="zephyr_openamp_rsc_table"
 CONFIG_PRINTK=n
 CONFIG_IPM=y
-CONFIG_IPM_MCUX=y
+CONFIG_IPM_IMX_MAX_DATA_SIZE_4=y
```

Comment:
This could be moved to `samples/subsys/ipc/openamp_rsc_table/boards/mimx8mm_evk.conf`.